index.js

```diff
@@ -50,10 +50,10 @@ var _outputCache = {
 
     helpers: {
 
-        setHeadersOnCacheItem: function onSetHeaders(req, res, cacheItem, headers) {
+        setHeadersOnCacheItem: function onSetHeaders(req, res, cacheItem) {
 
-            cacheItem.headers = headers;
-            var responseCacheHeader = headers['cache-control'];
+            cacheItem.headers = JSON.parse(JSON.stringify(res._headers));
```

Comment:
Yep, understood. However, parsing, creates a deep clone which avoid values changing later in the response.

Without the cloned object, Express does not send back the correct headers from the original response.